pkgs/applications/video/obs-studio/v4l2sink.nix

```diff
@@ -1,42 +1,49 @@
-{ stdenv, fetchFromGitHub
-, cmake, pkgconfig, wrapQtAppsHook
-, obs-studio }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, qt5
+, obs-studio
+}:
 
-stdenv.mkDerivation {
-  pname = "obs-v4l2sink-unstable";
-  version = "20181012";
+stdenv.mkDerivation rec {
```

Comment:
If this should probably be `mkDerivation` (fed by the argset), if `libsForQt5.callPackage` is used, although I'm not sure if `libsForQt5.callPackage` is necessary in the first place.

Thoughts?

Reply:
According to #76433, the qt-specific `mkDerivation` will automatically supply `wrapQtAppsHook`.

Reply:
I believe Qt's `mkDerivation` does more harm than good and should not have been introduced. Adding the dependencies (and `wrapQtAppsHook` for programs) should be sufficient. (There is also one minor improvement but it does not outweigh the loss of compositionality.)

Reply:
I'll keep it as-is then.